### PR TITLE
Orchestrator followup

### PR DESCRIPTION
**Pull request by taquitos**

### CODE REVIEW

File: PurchasesTests/Purchasing/PurchasesTests.swift [hunk 1]
```diff
@@ -2477,6 +2477,24 @@ class PurchasesTests: XCTestCase {
         }
     }
 
+    func testRetrievingDiscountMissingSubscriptionGroupIdentifierErrors() {
```
Comment (taquitos) [inline PurchasesTests/Purchasing/PurchasesTests.swift:2480]: The test

File: PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift [hunk 2]
```diff
@@ -35,4 +35,6 @@ import Foundation
         "a product identifier - this is an issue with the App Store." }
     @objc public var could_not_purchase_product_id_not_found: String { "makePurchase - Could not purchase SKProduct. " +
         "Couldn't find its product identifier. This is possibly an App Store quirk." }
+    var product_identifier_nil: String {"Apple returned a product where the productIdentifier is nil"}
+    var payment_identifier_nil: String {"Apple returned a payment where the productIdentifier is nil"}
```
Comment (aboedo) [inline PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift:39]: "This is possibly an App Store quirk"

File: PurchasesCoreSwift/Purchasing/PurchasesOrchestrator.swift [hunk 3]
```diff
@@ -119,6 +119,11 @@ public typealias RCDeferredPromotionalPurchaseBlock = (@escaping PurchaseComplet
             return
         }
 
+        guard let subscriptionGroupIdentifier = product.subscriptionGroupIdentifier else {
```
Comment (taquitos) [inline PurchasesCoreSwift/Purchasing/PurchasesOrchestrator.swift:122]: Here's what I was thinking @aboedo

**Review by aboedo — APPROVED** [review]

File: PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift (hunk 2 above)
Comment (taquitos) [reply PurchasesCoreSwift/Logging/Strings/PurchaseStrings.swift]: ✅

**Pull request merged**